lib/sshkit/ssh/channel.ex

```diff
@@ -266,4 +266,8 @@ defmodule SSHKit.SSH.Channel do
   end
 
   defp ljust(_, _), do: :ok
+
+  defp ssh_connection_module(conn) do
```

Comment:
May be a bit too early, but at some point one should probably align the usage of `conn` vs. `connection` in SSHKit. Right now both, the `Channel` and `Connection` modules already start mixing this in their function definitions - may become harder to understand for future users / contributers...